Lib/test/test_subprocess.py

```diff
@@ -1581,8 +1600,8 @@ class PopenNoDestructor(subprocess.Popen):
         def __del__(self):
             pass
 
-    @mock.patch("subprocess._posixsubprocess.fork_exec")
-    def test_exception_errpipe_normal(self, fork_exec):
+    @mock.patch(mock_modname)
+    def test_exception_errpipe_normal(self, mock_func):
```

Comment:
by abstracting the function name away from this code and going with the opaque mock_XXX names, it is no longer obvious what this does.  what function is stubbed out, what it does, etc.  It'd be good to keep this parameter named mock_fork_exec_or_spawn just to keep that clear to the reader without bouncing around the code to find definitions.

similar `mock_modname` would be better off called mock_fork_exec_or_spawn_fn_name for readability.